src/main/java/org/prebid/server/handler/NotificationEventHandler.java

```diff
@@ -0,0 +1,148 @@
+        }
+    }
+
+    @Override
+    public void handle(RoutingContext context) {
+        final EventNotificationRequest eventNotificationRequest;
+        try {
+            eventNotificationRequest = makeEventRequest(context.getBody());
+        } catch (InvalidRequestException ex) {
+            respondWithBadStatus(context, ex.getMessage());
+            return;
+        }
+
+        final NotificationEvent notificationEvent = new NotificationEvent(eventNotificationRequest.getType(),
+                eventNotificationRequest.getBidId(), eventNotificationRequest.getBidder());
+
+        analyticsReporter.processEvent(notificationEvent);
+
+        final Map<String, String> queryParams = HttpContext.from(context).getQueryParams();
+        final String format = queryParams.get(FORMAT_PARAMETER);
+
+        try {
+            validateEventRequestQueryParams(format);
+        } catch (InvalidRequestException ex) {
+            respondWithBadStatus(context, ex.getMessage());
+            return;
+        }
+
+        respondWithOkStatus(context, format);
+    }
+
+    private EventNotificationRequest makeEventRequest(Buffer body) {
+        if (body == null) {
+            throw new InvalidRequestException(
+                    "Request body was empty. Expected request with body has next fields: type, bidid and bidder.");
+        }
+
+        final EventNotificationRequest eventNotificationRequest;
+        try {
+            eventNotificationRequest = Json.decodeValue(body, EventNotificationRequest.class);
+        } catch (DecodeException e) {
+            throw new InvalidRequestException(
+                    "Request body couldn't be parsed. Expected request body has next fields: type, bidid and bidder.");
+        }
+
+        validateEventRequestBody(eventNotificationRequest);
+        return eventNotificationRequest;
+    }
+
+    private void validateEventRequestBody(EventNotificationRequest eventNotificationRequest) {
+        final String type = eventNotificationRequest.getType();
+        if (ObjectUtils.notEqual(type, VIEW_TYPE) && ObjectUtils.notEqual(type, WIN_TYPE)) {
+            throw new InvalidRequestException(String.format(
+                    "Type is required parameter. Possible values are win and view, but was %s", type));
+        }
+
+        final String bidId = eventNotificationRequest.getBidId();
+        if (StringUtils.isBlank(bidId)) {
+            throw new InvalidRequestException("bidid is required and can't be empty.");
+        }
+
+        final String bidder = eventNotificationRequest.getBidder();
+        if (StringUtils.isBlank(bidder)) {
+            throw new InvalidRequestException("bidder is required and can't be empty.");
+        }
+
+    }
+
+    private void validateEventRequestQueryParams(String format) {
+        if (format != null && ObjectUtils.notEqual(format, JPG) && ObjectUtils.notEqual(format, PNG)) {
```

Comment:
Starting from here and onwards: this kind of conditions is not scalable enough and may bite you when third or fourth format will be added, especially this one and alike: `format.equals(JPG) ? JPG_HEADER : PNG_HEADER`.
I suggest creating a map holding all the necessary information about supported format:
```
final Map<String, TrackingPixel> trackingPixels;
...
private static class TrackingPixel {
    String contentType;
    byte[] content
}
```

It will be easy to use it then for validation and response construction and it will be much safer when it will come to adding another format/